### PR TITLE
Content creation and development

### PR DESCRIPTION
**Pull request by Mogle7Arkad**
This pull request closes issues #5 .

### CODE REVIEW

**Comment by tennjugu** [comment]
Great job! Your attention to detail didn't go unnoticed—really loving the way you nailed it. Nice work!

File: dist/css/styles.css
Comment (Bazzy09) [inline dist/css/styles.css:1]: you did a great job, kudos.

**Review by Bazzy09 — APPROVED** [review]

File: dist/index.html
Comment (Bazzy09) [inline dist/index.html:1]: clear naming conventions for the element and the classes.

**Review by Bazzy09 — APPROVED** [review]
your work is well-structured and organized, with clear naming conventions for the different elements and classes used. The use of image tags for the movie posters and buttons for actions like commenting and making reservations is also a good design choice. well-done sir.

File: dist/css/styles.css
Comment (Mogle7Arkad) [reply dist/css/styles.css:1]: thank you sir.

**Comment by Mogle7Arkad** [comment]
thank you @tennjugu  and @Bazzy09  for the reviews.

**Pull request merged**